docs/reference/remap/length.cue

```diff
@@ -0,0 +1,47 @@
+package metadata
+
+remap: functions: length: {
+	arguments: [
+		{
+			name:        "value"
+			description: "The array or map"
+			required:    true
+			type: ["array", "map"]
+		},
+	]
+	internal_failure_reason: null
+	return: ["integer"]
+	category: "Enumerate"
+	description: """
+		Return the length of the array or the number of keys in the map (nested keys are ignored).
+		"""
+	examples: [
+		{
+			title: "Standard map"
+			input: log: teams: {
+				portland: "Trail Blazers"
+				seattle:  "Supersonics"
+			}
+			source: ".num_teams = length(del(.teams))"
```

Comment:
Not a big deal, but I might have the examples _just_ use `length` rather than `del` too. I feel like using the both like you have here would not be terribly common.

Reply:
Yeah, I suspect you're right